rasa/core/actions/action.py

```diff
@@ -1045,10 +1040,7 @@ async def _run_custom_action(
                     disallowed_types.add(event.type_name)
         except (RasaException, ClientResponseError) as e:
             logger.warning(
-                f"Failed to execute custom action '{custom_action}' "
-                f"as a result of error '{str(e)}'. The default action "
-                f"'{self.name()}' failed to fill slots with custom "
-                f"mappings."
+                f"Failed to execute custom action '{custom_action}' as a result of error '{e}'. The default action '{self.name()}' failed to fill slots with custom mappings."
```

Comment:
Function `ActionExtractSlots._run_custom_action` refactored with the following changes:

- Simplify unnecessary nesting, casting and constant values in f-strings ([simplify-fstring-formatting](https://docs.sourcery.ai/refactorings/simplify-fstring-formatting/))